PelotonEppSdkTests/BankAccountCreateTests.cs

```diff
@@ -20,7 +20,7 @@ public void TestCreateAccount()
             createRequest.BankAccount.Token = Guid.NewGuid().ToString().Substring(0, 32);
```

Comment:
Even though this is a test, we typically strip the hyphens from the Guid to end up with a unique identifier that is 32 characters long. .replace("-","")